packages/web/components/swap-tool/index.tsx

```diff
@@ -722,16 +724,21 @@ export const SwapTool: FunctionComponent<SwapToolProps> = observer(
                       showPriceImpactWarning ? "opacity-100" : "opacity-0"
                     )}
                   />
-                  <Icon
-                    id="chevron-down"
-                    height={isMobile ? 14 : 18}
-                    width={isMobile ? 14 : 18}
-                    className={classNames(
-                      "text-osmoverse-400 transition-all",
-                      showQuoteDetails ? "rotate-180" : "rotate-0",
-                      isQuoteDetailRelevant ? "opacity-100" : "opacity-0"
-                    )}
-                  />
+                  {
+                    // if unavailalble, hide the icon
+                    isSwapAssetsSpotPriceAvailable && (
```

Comment:
When spot price is unavailable, this icon would be in an awkward place so made it hidden